libs/common-utils/src/explorer.ts

```diff
@@ -50,6 +51,7 @@ enum Explorers {
   Explorer = 'Explorer',
   Blockscout = 'Blockscout',
   Etherscan = 'Etherscan',
+  Arbiscan = 'Arbiscan',
 }
 
 // Used for GA ExternalLink detection
```

Comment:
The function is only used in `externalLinkAnalytics`, I'm curious, do we still need the event?

Reply:
I think so?

![image](https://github.com/cowprotocol/cowswap/assets/43217/153d781a-a651-419d-8267-e791fd0719fd)

Not 100% sure